packages/drawer/src/react/index.js

```diff
@@ -0,0 +1,155 @@
+import 'element-closest'
+import Collapsible from './collapsible';
+import core from '@pluralsight/ps-design-system-core'
+import * as glamor from 'glamor'
+import glamorous, { Div } from 'glamorous'
+import { transparentize } from 'polished'
+import Icon, { sizes as iconSizes } from '@pluralsight/ps-design-system-icon/react'
+import { names as themeNames } from '@pluralsight/ps-design-system-theme/react'
+import { bool, func, node, string } from 'prop-types'
+import React from 'react'
+
+const { gray01, gray02, gray03, gray04, gray06, white } = core.colors
+
+const DrawerBase = glamorous.div(
+  {
+    position: 'relative',
+    cursor: 'pointer',
+    transition: `background ${core.motion.speedNormal}`,
+    ':hover': {
+      background: transparentize(0.9, gray02)
+    }
+  },
+  ({ isOpen }) => isOpen
+    ? {
+      background: transparentize(0.8, gray02),
+      ':hover': {
+        background: transparentize(0.8, gray02)
+      }
+    }
+    : {}
+)
+
+const DrawerPanel = glamorous.div(
+  {
+    background: transparentize(0.94, gray02),
+    transition: `box-shadow ${core.motion.speedNormal}`
+  },
+  ({ isOpen, themeName }) =>(isOpen
+    ? { boxShadow: `inset 0 1px 3px 0 rgba(0,0,0,${themeName === themeNames.dark ? '0.5' : '0.1'})` }
+    : {}
+  )
+)
+
+const RowContainerWrapper = glamorous.div(
+  ({ themeName }) => ({
+    borderTop: `1px solid ${themeName === themeNames.dark ? gray04 : gray01 }`
+  })
+)
+const RowContainer = (props, context) => (
```

Comment:
Why the design choice to tie Row to the drawer like this?  Could it be solved in a more general way?

Reply:
There are a couple display issues that show up only when adding a `Row` component inside the `Drawer`.
- The borders of the row don't extend far enough, so we have to add our own border. In order to not double-up on borders we have to shift the row up a single pixel.
- The padding on the left of the DrawerBase is really only necessary when using a `Row`. So we're using RowContainer to add that spacing.

Basically, we're planning on saying that if you need to use a `Row` in the `Drawer`, you should wrap it in a `RowContainer`. Otherwise you won't need to use it.

We thought this was the best solution for the consumer but would love to hear your feedback and suggestions..

Reply:
We chatted as a group and decided to make the Drawer support the Row.  We don't know what else it should/will support someday.  We'll optimize the primary case.  RowContainer should be able to go away.  Margin -1 still needs to happen.